csrankings-h.csv

```diff
@@ -308,7 +308,7 @@ Hao Huang 0001,Wuhan University,https://cs.whu.edu.cn/info/1019/2467.htm,yBUENP0
 Hao Li 0011,Xi'an Jiaotong University,https://aquatoney.github.io,-rpSVLEAAAAJ
 Hao Li 0015,MBZUAI,https://mbzuai.ac.ae/study/faculty/hao-li,NFeigSoAAAAJ
 Hao Liao,Shenzhen University,https://csse.szu.edu.cn/pages/user/index?id=542,Tu5ZuREAAAAJ
-Hao Su 0001,Univ. of California - San Diego,http://ai.ucsd.edu/~haosu,1P8Zu04AAAAJ
+Hao Su 0001,Univ. of California - San Diego,https://cseweb.ucsd.edu/~haosu/,1P8Zu04AAAAJ

```

Comment:
Maybe in industry: Chairman & CTO of [Hillbot Inc.](https://www.hillbot.ai/)

Reply:
Timeline looks like sabbatical leave.